install/kubernetes/cilium/values.yaml

```diff
@@ -1643,3 +1643,10 @@ clustermesh:
 externalWorkloads:
   # -- Enable support for external workloads, such as VMs (false by default).
   enabled: false
+
+# -- Configure cgroup root where cgroup v2 filesystem is mounted
+cgroupRoot: /run/cilium/cgroupv2
```

Comment:
I don't understand this one. With the current version of the `master` branch, the cgroupv2 are already available in this directory. Why do we need to specify it as an helm flag?

Reply:
This helm value corresponds to the Cilium agent config option (`cgroup-root`) that allows users to specify a path where cgroup v2 is mounted.